Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2369 -- Change repository sidebar to not be sticky

Resolves [EREGCSC-2369](https://jiraent.cms.gov/browse/EREGCSC-2369)

**Description**

The left sidebar on the Policy Repository page had been styled as `position: sticky`, which meant that the sidebar filters would always be visible, no matter if you scrolled to the bottom of a long list of results or not.

Based on feedback from user research sessions, the sidebar should no longer be sticky positioned, and should instead scroll out of view along with the results list.

**This pull request changes:**

- Removes styles and markup associated with making the left sidebar sticky positioned
- Tweaks sidebar width so that it no longer overflows into the results content at around 770px/`md` widths

**Steps to manually verify this change:**

1. Visit the [policy repository on the experimental deployment](https://rrzj783vrg.execute-api.us-east-1.amazonaws.com/dev1072/policy-repository/)
2. View a long list of results (e.g. select Public resources only)
3. Scroll down to the bottom of the results list
4. Notice that the left sidebar filters scroll up and out of view along with the results
